cmds/boot/systemboot/main.go

```diff
@@ -91,11 +371,21 @@ func runIPMICommands() {
 	if productName, err := getSystemProductName(si); err == nil {
 		if isMatched(productName) {
 			log.Printf("Running OEM IPMI commands.")
+			if err := IpmiOemSetProcessorAndDimmInfo(si); err == nil {
```

Comment:
If you are already sending the info in line 384, is this needed here?

Reply:
It`s redundant, removed it.